test/cases/arel/nodes_test.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
```

Comment:
This can be removed on next rails update as it was fixed upstream

Reply:
could a comment be added that mentions this?

Reply:
There is also one (see the excluded folder, it is mentionning the given PR). But I can do more for sure :)